drivers/pulse_counter/include/pulse_counter_params.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2017 UC Berkeley
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_pulse_counter
+ *
+ * @{
+ * @file
+ * @brief       Default configuration for PULSE_COUNTER devices
+ *
+ * @author      Hyung-Sin Kim <hs.kim@cs.berkeley.edu>
+ */
+
+#ifndef PULSE_COUNTER_PARAMS_H
+#define PULSE_COUNTER_PARAMS_H
+
+#include "board.h"
+#include "pulse_counter.h"
+#include "saul_reg.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef PULSE_COUNTER_PARAMS
+#define PULSE_COUNTER_PARAMS       { .gpio = GPIO_PIN(PA,18), \
```

Comment:
`GPIO_PIN(PA,18)` could defined in a seperate default macro before:
```c
#ifndef PULSE_COUNTER_GPIO
#define PULSE_COUNTER_GPIO          GPIO_PIN(PA,18)
#endif
```
same for `.gpio_flank`. 

and then here:
```c
#define PULSE_COUNTER_PARAMS       { .gpio = PULSE_COUNTER_GPIO, \
...
```
The idea is that if a board has a pulse counter then one has just to define `PULSE_COUNTER_GPIO` in `board.h`

Reply:
This comment has not been addressed.